.devcontainer/devcontainer.json

```diff
@@ -14,9 +14,12 @@
     "mounts": [
         "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind"
     ],
-    "initializeCommand": "bash .devcontainer/build.sh --tag matter-dev-environment:local --version 22",
+    "initializeCommand": "bash .devcontainer/build.sh --tag matter-dev-environment:local --version 74",
     "image": "matter-dev-environment:local",
     "remoteUser": "vscode",
+    "containerEnv": {
+        "PW_ENVIRONMENT_ROOT": "/workspaces/connectedhomeip/.environment-vscode"
```

Comment:
Add this to .gitignore, preferably as `/.environment*`

Reply:
Done.